### ISSUE TITLE
Returns false for passwords found once

### ISSUE THREAD

**Issue by nmxcgeo** [ron-maxweb/pwned-passwords#1]
In your readme.md you write: 

> The `isInsecure` method will return true if the password has been found in the PwnedPasswords API, and false if not.

However due to the option `$maxUsage = 1` and 
```
               if (intval($passwordLine[1]) > $maxUsage) {
                    return true;
                }
```

(n.b. you used greater) the function returns false for passwords found exactly once in the API. 

I tested it using the hash `ABCDE028DA52CF202EA546E9A7669C7614D64DCD`, but I don't know what password it's for (i just found it having one appearance for `ABCDE` :-)).

To fix that I suggest to set the default value for  `$maxUsage` to 0 in order to comply with the function documentation, the readme and the naming of `$maxUsage` (which to me implies that a password occurring exactly `$maxUsage` times should return false).

Greetings
Nmxcgeo

**Comment by azjezz** [comment]
pull request #4 should fix this

**Comment by azjezz** [comment]
@ron-maxweb this one should be closed too, fixed !

**Comment by mxrxdxn** [comment]
Fixed in #4.
